### PR TITLE
get the version from the package.json

### LINKED ISSUE #199: ts2fable@next does not print version on run
### ts2fable@next
```
C:\Users\camer\fs\ts2fable [shebang ≡]> ts2fable --version
0.6.0-build.289
C:\Users\camer\fs\ts2fable [shebang ≡]> ts2fable node_modules/@types/react/index.d.ts test-compile/React.fs
ts2fable 0.0.0
```

### ts2fable
```
C:\Users\camer\fs\ts2fable [shebang ≡]> ts2fable --version
0.5.2
C:\Users\camer\fs\ts2fable [shebang ≡]> ts2fable node_modules/@types/react/index.d.ts test-compile/React.fs
ts2fable 0.5.2
```

### PR DESCRIPTION
**Pull request by ctaggart**
fix #199

### CODE REVIEW

**Comment by humhei** [comment]
We doesn't invoke `version.fs`
But the mocho will auto invoke it
And `___dirname` will be incorrent that which will throw a exception

File: src/version.fs [hunk 1]
```diff
@@ -8,6 +8,6 @@ type PackageJson =
         version: string
     }
 let version =
```
Comment (humhei) [inline src/version.fs]: Change 
`let version` to `let version()` may get it worked

Comment (humhei) [reply src/version.fs]: `Try with` pattern may also take effect

**Comment by humhei** [comment]
@ctaggart  
Official coming back?
Still busy?

**Comment by ctaggart** [comment]
> Official coming back?

May be, not sure yet. I'm in between job contracts at the moment. It depends if I can use Fable for my next one like I'm hoping.

**Comment by ctaggart** [comment]
Version fixed:
```
C:\Users\camer\fs\ts2fable [master ≡]> ts2fable node_modules/@types/react/index.d.ts test-compile/React.fs
ts2fable 0.6.0-build.294
```

**Pull request merged**